test/integration/zk-discovery.bats

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bats
+
+load helpers
+
+ZK_HOST=127.0.0.1:2181
+
+function start_zk() {
+	docker run --name zk -p $ZK_HOST:2181 -d jplock/zookeeper
```

Comment:
Instead of hardcoding `zk`, perhaps it should be a variable declared above.

Maybe something longer `swarm_integration_zk` to avoid conflicts?